recordscreen.py

```diff
@@ -238,22 +238,13 @@ def get_default_output_path(ext=None):
 def print_codecs():
     """ Prints a list of the available audio/video codecs.
     """
-    a = []
-    v = []
-    for i in acodecs:
```

Comment:
Looking at this again I wonder if the functionality you were actually looking for was:

```
for i in acodecs.values():
    a += i
```

If this is the case pls let me know and I will push the correct fix.